APE18.rst

```diff
@@ -0,0 +1,69 @@
+Adopt NEP 29 for CPython and Numpy Version Support
+--------------------------------------------------
+
+author: Stuart Mumford and Thomas Robitaille
+
+date-created: 2020 October 16
+
+date-last-revised: 2020 October 16
+
+type: Standard Track
+
+status: Discussion
+
+
+Abstract
+--------
+
+After the release of astropy 4.0 APE 10 no longer specifies the supported versions of CPython.
```

Comment:
Technically APE 10 does provide a recipe for future versions. Maybe just say that APE 10 previously determined the versions of CPython we supported and provided a specific list up to 4.0 LTS or something like that?